assets/ant-changelog.md

```diff
@@ -0,0 +1,51 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+<!-- eslint-disable-next-line -->
+
+## [7] - [p6svP-fOm1N9imdx9wF2h3u81Qbc7y-HLEGBDCCE3s4] - (2024-10-03)
```

Comment:
nit - add links to arweave

Reply:
Not sure what you mean here, like in the changelog from the ant repo? because this actually gets parsed in the modal to create a link with the id.